test/models/user_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```

Comment:
same here, for now we will delete the unused files. If needed in future will create.